R/gtsave.R

```diff
@@ -78,22 +93,58 @@ gt_save_html <- function(data, filename, ..., inline_css = FALSE) {
   }
 }
 
-# Saving function for a LaTeX file
+#' Saving function for a LaTeX file
+#'
+#' @importFrom htmltools HTML save_html
+#' @noRd
+gt_save_webshot <- function(data, filename, ...) {
+
+  tempfile_ <- tempfile(fileext = ".html")
+
+  data %>%
```

Comment:
Wouldn't this be better as `data %>% htmltools::as.tags() %>% htmltools::save_html(...)`? I think this would allow html dependencies that you might want to bring along, to be included.

Reply:
Yes, I agree. Changed this to use the `gt_save_html()` function (with the default option to *not* inline the HTML styles this time). This call performs the following:

```r
data %>%
  htmltools::as.tags() %>%
  htmltools::save_html(filename)
```